django_ipam/management/commands/__init__.py

```diff
@@ -0,0 +1,45 @@
+import csv
+from ipaddress import ip_network
+
+from django.core.management.base import BaseCommand, CommandError
+
+
+class BaseImportSubnetCommand(BaseCommand):
+    help = 'Import an entire Subnet data from a csv/xls/xlsx file'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--file', type=str, help="File path to import csv data.")
+
+    def handle(self, *args, **options):
+        file = options['file']
+        try:
+            if not file.endswith(('.csv', '.xls', '.xlsx')):
+                raise CommandError('File type not supported')
+            with open(file, 'rb+') as csvfile:
+                if not self.subnet_model.import_csv(self, csvfile):
```

Comment:
as explained before, here you should catch our custom exception and show the details about what went wrong to the user